### PR TITLE
Enables pdf worker

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

HOTT-<TODO>

### What?

I have added/removed/altered:

- [x] Enable PDF worker
- [x] Execute worker with currency of current service

### Why?

I am doing this because:

- We need to verify the behaviour of the pdf workers and want them enabled eventually

### CODE REVIEW

File: config/sidekiq.yml [hunk 1]
```diff
@@ -33,7 +33,6 @@
   ReindexModelsWorker:
     cron: "0 1 * * *" # 01:00 every day
     description: "ReindexModelsWorker will run every day at 1am."
-  # RunChapterPdfWorker:
-  #   cron: "0 5 * * *" # 05:00 every day
-  #   description: "RunChapterPdfWorker produces the Tariff PDF."
-  #   args: [['EUR', 'GBP']]
+  RunChapterPdfWorker:
+    cron: "0 5 * * *" # 05:00 every day
+    description: "RunChapterPdfWorker produces the Tariff PDF."
```
Comment (tavy88) [inline config/sidekiq.yml:38]: Can we make sure that the lack of args still works? Should we remove the args from the worker now?

Comment (willfish) [reply config/sidekiq.yml:38]: It does, tested locally

File: app/workers/run_chapter_pdf_worker.rb [hunk 2]
```diff
@@ -5,28 +5,24 @@ class RunChapterPdfWorker
...
-        batch.jobs do
-          GenerateCoverPdfWorker.perform_async(currency)
-        end
-        batch.jobs do
-          chapter_ids.shuffle.each do |sid|
-            GenerateChapterPdfWorker.perform_async(sid, currency)
-          end
+      batch.jobs do
+        chapter_ids.shuffle.each do |sid|
+          GenerateChapterPdfWorker.perform_async(sid, currency)
```
Comment (tavy88) [inline app/workers/run_chapter_pdf_worker.rb:25]: Unit tests?

Comment (willfish) [reply app/workers/run_chapter_pdf_worker.rb:25]: Spoken about this in person. We don't currently have any tests for this file. Going to add a story to backfill these tests as part of the ongoing changes to the pdf generation

**Review by tavy88 — APPROVED** [review]

**Pull request merged**